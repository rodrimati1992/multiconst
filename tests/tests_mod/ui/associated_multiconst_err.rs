
// ensuring that the macro doesn't work outside of inherent impls.
multiconst::associated_multiconst!{
    const [A, B]: [u32; 2] = [3, 5];
}

fn main(){}