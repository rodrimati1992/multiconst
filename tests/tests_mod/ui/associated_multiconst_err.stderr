error[E0433]: failed to resolve: use of undeclared type `Self`
 --> tests/tests_mod/ui/associated_multiconst_err.rs:3:1
  |
3 | / multiconst::associated_multiconst!{
4 | |     const [A, B]: [u32; 2] = [3, 5];
5 | | }
  | |_^ use of undeclared type `Self`
  |
  = note: this error originates in the macro `$crate::__::__priv_associated_multiconst_proc_macro` (in Nightly builds, run with -Z macro-backtrace for more info)
